stdlib/ballerina-database/src/main/ballerina/jdbc/Package.md

```diff
@@ -179,9 +179,9 @@ match ret5 {
 
 ### Calling stored procedures
 
-Following are three examples of executing stored procedures via the  `call` operation of the endpoint. 
+The following examples demonstrate executing stored procedures via the  `call` operation of the endpoint. 
```

Comment:
There is an extra space before `call`.

Reply:
Fixed with this commit: https://github.com/ballerina-platform/ballerina-lang/pull/8071/commits/b100133f6879d70445f59461cd7a1c84ecc1c05b